src/coreclr/vm/gcinfodecoder.cpp

```diff
@@ -19,7 +19,11 @@
 
 
 #ifndef GET_CALLER_SP
-#define GET_CALLER_SP(pREGDISPLAY) EECodeManager::GetCallerSp(pREGDISPLAY)
+inline size_t GET_CALLER_SP(PREGDISPLAY pREGDISPLAY)
+{
```

Comment:
Is this method actually used? It looks like a left-over from JIT64 era. Can we turn it into `ASSERT(false); return NULL;` or something like that?

Reply:
I've added the ASSERT here and it hasn't fired while running coreclr / libraries tests, so it looks like it is really obsolete. I'll commit that change.